app/views/shared/_header.html.erb

```diff
@@ -1,3 +1,2 @@
 <%= render "shared/header/masthead" %>
-<%= render "shared/header/nav" %>
```

Comment:
what does this do?

Reply:
So currently in production /nav is the 2nd row nav menu that appears on smaller devices. In order for the fader to work for the menu, I had to remove the call to that 2nd nav and adjust the main centered nav to stay visible but move beneath the logo.

Adding the fader to that /nav menu didn't work as the two rows being separate containers wouldn't be in sync. Plus this way adds the benefit of being DRY-compliant since there will no longer be a duplicated header menu in the DOM just waiting for a media query to kick in.